cache.go

```diff
@@ -51,7 +50,7 @@ var _ QuotaCache = (*RedisCache)(nil)
 var _ QuotaCache = (*LRU)(nil)
 var _ UsageCache = (*RedisCache)(nil)
 
-func NewLimitCounter(cfg RedisConfig, logger logger.Logger) httprate.LimitCounter {
+func NewLimitCounter(cfg RedisConfig, log *slog.Logger) httprate.LimitCounter {
```

Comment:
i think its too soon for this unfortunately because not all of our services use slog yet, but goware/logger supports both slog and zerolog. One day we will do this.. but we need to get everything to slog first.